### PR TITLE
gh-105545: Remove deprecated `MacOSXOSAScript._name`

### PR DESCRIPTION
**Pull request by sobolevn**
* Issue: gh-105545

### CODE REVIEW

File: Doc/whatsnew/3.13.rst [hunk 1]
```diff
@@ -324,6 +324,9 @@ Removed
   certificates for you.
   (Contributed by Victor Stinner in :gh:`105382`.)
 
+* Remove deprecated ``webbrowser.MacOSXOSAScript._name`` attribute.
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst:327]: Shall we also mention the replacement?

Comment (sobolevn) [reply Doc/whatsnew/3.13.rst:327]: Yes, sure!

File: Doc/whatsnew/3.13.rst [hunk 2]
```diff
@@ -324,6 +324,10 @@ Removed
...
+  Use ``.name`` attribute instead.
```
Comment (corona10) [inline Doc/whatsnew/3.13.rst]:
```suggestion
  Use :attr:`webbrowser.MacOSXOSAScript.name` attribute instead.
```

File: Doc/whatsnew/3.13.rst [hunk 3]
```diff
@@ -324,6 +324,10 @@ Removed
   certificates for you.
   (Contributed by Victor Stinner in :gh:`105382`.)
 
+* Remove deprecated ``webbrowser.MacOSXOSAScript._name`` attribute.
+  Use :attr:`webbrowser.MacOSXOSAScript.name` attribute instead.
```
Comment (corona10) [inline Doc/whatsnew/3.13.rst]: Sorry would you like to link to this attribute? https://docs.python.org/3.13/library/webbrowser.html#webbrowser.name

Comment (sobolevn) [reply Doc/whatsnew/3.13.rst]: Done!

**Review by corona10 — APPROVED** [review]
LGTM!

**Pull request merged**